install flow i18n (Fixes #600)

samlanning opened the pull request:
First attempt at adding i18n to the install flow.

I haven't fully tested #aa86555 due to lack of a free android device, but the previous two commits have been fully tested.

Fixes #600

samlanning commented on js/options.js:
```diff
@@ -58,4 +58,31 @@
+            href: 'https://twitter.com/whispersystems',
+            target: '_blank'
+        });
+        $('#installFollowUs').append($installFollowUsContent);
```
We may want to refactor out some of the common code here, perhaps by creating another function / extending `i18n` to allow for attribute replacement, and then generate the html rather than a jQuery object.

Perhaps usage could be something like this?

```javascript
var installFollowUsHtml = i18n('installFollowUs', {
    // any jQuery selector
    'a': {
        class: 'link',
        href: 'https://twitter.com/whispersystems',
        target: '_blank'
    }
});
$('#installFollowUs').html(installFollowUsHtml);
```

liliakai replied:
You might be able to solve this elegantly by using [placeholders](https://developer.chrome.com/extensions/i18n-messages#placeholders) and/or [substitution strings](https://developer.chrome.com/extensions/i18n#method-getMessage). Our i18n wrapper should certainly be extended to expose this feature if it proves useful, but I would try keep any additional rendering logic encapsulated in the view class.

liliakai commented on js/options.js:
```diff
@@ -58,4 +58,31 @@
+
+    // Apply i18n
+    $(document).ready(function(){
+        // Basic Substitution
```
Rather than jquery all the things, we should probably convert most of the html markup for this page to a Mustache template that the InstallView can render with i18n'd content.

samlanning commented:
Addressed your comments. Is this alright?

It's probably worthwhile squashing some of these commits together now though... I'll leave it as is for the time being for reviewing purposes. =)

And the diff is much nicer with [?w=1](611/files?w=1)

samlanning commented:
did not mean to press that button... :grimacing:

liliakai commented:
lg2m, go ahead and squash

samlanning commented:
Done. Squashed into 2 commits =)

liliakai commented:
ac25b62
0825d3e
